azurerm/resource_arm_app_service_source_control_token.go

```diff
@@ -0,0 +1,119 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/services/web/mgmt/2018-02-01/web"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/features"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/locks"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+var appServiceSourceControlTokenResourceName = "azurerm_app_service_source_control_token"
+
+func resourceArmAppServiceSourceControlToken() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmAppServiceSourceControlTokenCreateOrUpdate,
```

Comment:
Could we use `CreateUpdate` for consistency with other resources?
```suggestion
		Create: resourceArmAppServiceSourceControlTokenCreateOrUpdate,
```